execution/src/tests/scenarios_mandatories.rs

```diff
@@ -5,6 +5,28 @@ use serial_test::serial;
 
 #[tokio::test]
 #[serial]
-async fn test_basic_transaction() {
-    assert!(start_controller(ExecutionConfig {}).await.is_ok());
+async fn test_execution_basic() {
+    assert!(start_controller(ExecutionConfig {}, 2).await.is_ok());
+}
+
+#[tokio::test]
+#[serial]
+async fn test_execution_shutdown() {
+    let (_command_sender, _event_receiver, manager) = start_controller(ExecutionConfig {}, 2)
+        .await
+        .expect("Failed to start execution.");
+    manager.stop().await.expect("Failed to stop execution.");
+}
+
+#[tokio::test]
+#[serial]
+async fn test_sending_command() {
```

Comment:
Is there a way to auto generate tests that are just asserting that a command is propagated, not only here but every where else ?

Reply:
you mean like a macro?

Reply:
I don't know, we could open an issue to discuss/investigate further.

Reply:
#1949